### ISSUE TITLE
[Meta] Full Audit

### ISSUE THREAD

**Issue by Garbee** [GoogleChrome/devtools-docs#152]
The entire documentation can use a run-through to gather as many issues as possible to be fixed. Once I get the next two PRs that I'm working on pulled and made then I'll work on this.

Steps to complete:

1. List every page in the docs.
1. Go through each page of documentation, even the latest content.
1. Make a new issue for each page with content issues.
    * list and detail what needs improving.
    * Split the list into two sections, critical and beneficial.
        * Critical - Major issues in the content. Either the flow is out of sync, images being used for primary content, or pointing out where things just don't make sense.
        * Beneficial - Minor things like changing unordered lists to definition lists. Not necessary to address content issues, but still nice improvements.
1. While auditing the pages, find a good primary fit for where they would go if they were listed in the top nav area. This will help us float some of the more obscure resources and tools to attention.

I'll update this PR with the list. Then as they are worked on, link to the new issues and mark them off. Once the audit is done and all that is left are individual issues, this is closed and the page-scoped issues are the focus.

**Comment by Garbee** [comment]
With the recent talk of moving a good chunk of the content out of DCC and refocusing on just Reference docs, I'm closing this since it will naturally be done in that movement.
